index.html

```diff
@@ -45,19 +45,18 @@
       <p>
         This specification defines the authoring rules (author conformance
         requirements) for the use of [[[wai-aria-1.1]]] and [[[dpub-aria-1.0]]]
-        attributes on [[HTML| HTML]] elements. This specification's primary objective is to define these requirements for use with Conformance Checking tools, where the intended recipients of such checking tools are authors (web developers). These requirements will aid authors in their development of custom web interfaces that do not entirely rely on the features of the host language [[HTML]].
+        attributes on [[HTML| HTML]] elements. This specification's primary objective is to define requirements for use with conformance checking tools used by authors (web developers). These requirements will aid authors in their development of web content (including custom interfaces/widgets) that makes use of ARIA to complement or extend the features of the host language [[HTML]].
```

Comment:
```suggestion
        attributes on [[HTML]] elements. This specification's primary objective is to define requirements for use with conformance checking tools used by authors (i.e., web developers). These requirements will aid authors in their development of web content, including custom interfaces/widgets, that makes use of ARIA to complement or extend the features of the host language [[HTML]].
```

Reply:
:+1:

Reply:
@marcoscaceres any idea why Travis might not be happy all of a sudden?